### DISCUSSION TITLE
docs/3.x/developer/extending/webhooks/synchronous-events/shipping

### DISCUSSION THREAD

**Discussion by giscus** [saleor/saleor-docs#1036] [General]
# docs/3.x/developer/extending/webhooks/synchronous-events/shipping

Introduction

https://docs.saleor.io/docs/3.x/developer/extending/webhooks/synchronous-events/shipping

**Comment by simon-online** [comment]
It would be great to have some more details on the best way for the webhook to determine the `<app-database-id>`.

Perhaps even some example code.

**Reply by timuric** [reply]
Hey Simon, could you please describe your business use case? Thanks!

**Reply by simon-online** [reply]
On this page it says:

External shipping methods have an ID of the following format: `app:<app-database-id>:<id-returned-by-webhook>`

But it doesn't explain where you can get `<app-database-id>` from eg. from decoding the recipient id in the payload.
